lib/knackhq/client.rb

```diff
@@ -0,0 +1,42 @@
+require 'knackhq/client/version'
+require 'blanket'
+require 'hashie'
+
+module Knackhq
+  class Client
+    attr_accessor :base_uri, :x_knack_application_id, :x_knack_rest_api_key
+
+    def initialize(base_uri, x_knack_application_id, x_knack_rest_api_key)
+      @base_uri = base_uri
+      @x_knack_application_id = x_knack_application_id
+      @x_knack_rest_api_key = x_knack_rest_api_key
+    end
+
+    def objects
```

Comment:
If you wanted to be clever you could return some kind of ClientQuery or ObjectQuery that then used method missing (methods matching object_*) for building object based queries?

Something to think about if you're feeling adventurous. :smile: